packages/bot-updategas/config/custom-environment-variables.json

```
@@ -12,6 +12,10 @@
     "__name": "RUN_EVERY_X_HOURS",
     "__format": "number"
   },
+  "overEstimateOracleGasPriceByXPercent": {
+    "__name": "OVERESTIMATE_ORACLE_GASPRICE_BY_X_PERCENT",
+    "__format": "nubmer"
```

```suggestion
    "__format": "number"
```

Should probably test that this works locally before merging.